lib/__tests__/test.js

```diff
@@ -2,7 +2,45 @@
 
 const fs = require('fs');
 const path = require('path');
-const defineTest = require('jscodeshift/src/testUtils').defineTest;
+const {walkDir} = require('../execution-helper');
+
+/**
+ * @param {string} dirname - the testing directory
+ * @param {string} transformName - the name of the transform
+ * @param {string} filePrefix - the test file prefix
+ */
+function defineTest(dirname, transformName, filePrefix) {
+  const testName = `rewrites correctly on ${filePrefix}`;
+  describe(transformName, () => {
+    it(testName, () => {
+      runTest(dirname, transformName, filePrefix);
+    });
+  });
+}
+
+/**
+ * @param {string} dirname - the testing directory
+ * @param {string} transformName - the name of the transform
+ * @param {string} filePrefix - the test file prefix
+ */
+function runTest(dirname, transformName, filePrefix) {
+  const fixtureDir = path.join(dirname, '..', '__testfixtures__');
+  const inputPath = path.join(fixtureDir, filePrefix + '.input.js');
+  const source = fs.readFileSync(inputPath, 'utf8');
+  const expectedOutput = fs.readFileSync(
+      path.join(fixtureDir, filePrefix + '.output.js'),
```

Comment:
Might as well have an `outputPath` for symmetry with `inputPath`, even if you only use it once.